src/gewis/gewis.ts

```diff
@@ -356,6 +356,12 @@ export default class Gewis {
         Balance: {
           get: { all: star },
         },
+        Event : {
+          get: { all: star },
+          update: { all: star },
+          delete: { all: star },
+          create: { all: star },
+        },
```

Comment:
You miss the "assign EventAnswer" rule. https://github.com/GEWIS/sudosos-backend/blob/e720d09bda53b81f9cdd2f9a28d43cd3cf740f29/src/controller/event-controller.ts#L89